src/status_ctrl.js

```diff
@@ -51,19 +51,20 @@ export class StatusPluginCtrl extends MetricsPanelCtrl {
 				if(angular.isNumber(input)) {
 					return this.filter('number')(input);
 				} else {
-
 					if(textRegex == null || textRegex.length == 0) {
 						return input;
-					}
-					else {
-						var regex = new RegExp(textRegex);
-						var matchResults = input.match(regex);
-						if(matchResults == null) {
+					} else {
+						try {
+							let regex = new RegExp(textRegex);
+							let matchResults = input.match(regex);
+							if (matchResults == null) {
+								return input;
+							} else {
+								return matchResults[0];
+							}
+						} catch (e) {
```

Comment:
Couldn't you just put the `try/catch` block around the line that throws the exception?
I think it will look much cleaner.